src/Namshi/Notificator/Notification/Email/SwiftMailer/SwiftMailerNotification.php

```diff
@@ -18,26 +18,15 @@ class SwiftMailerNotification extends EmailNotification implements NotificationI
     /**
      * Constructor.
      *
-     * @param string $emailTemplate
-     * @param string $recipientAddress
-     * @param array $parameters
+     * @param string        $emailTemplate
+     * @param array|string  $recipientAddress
+     * @param array         $parameters
      */
     public function __construct($emailTemplate, $recipientAddress, array $parameters = array())
     {
-        $this->setEmailTemplate($emailTemplate);
-        $this->setRecipientAddress($recipientAddress);
-        $this->setParameters($parameters);
-    }
-
+        parent::__construct($recipientAddress, $parameters);
```

Comment:
No, Its a string or array of addresses.
I will change the variable name to `$recipientAddresses`